tests/infra/consortium.py

```diff
@@ -389,12 +389,12 @@ def add_users_and_transition_service_to_open(self, remote_node, users):
         for user_id in users:
             cert = slurp_file(self.user_cert_path(user_id))
             proposal["actions"].append({"name": "set_user", "args": {"cert": cert}})
-        proposal["actions"].append(
-            {
-                "name": "transition_service_to_open",
-                "args": {"next_service_identity": self.get_service_identity()},
-            }
-        )
+
+        args = {}
+        if not remote_node.version_before("ccf-2.0.0-rc4"):
+            args = {"next_service_identity": self.get_service_identity()}
+        proposal["actions"].append({"name": "transition_service_to_open", **args})
```

Comment:
```suggestion
        proposal["actions"].append({"name": "transition_service_to_open", "args": args})
```

Reply:
This was correct - in case `args` is empty I want no "args" entry at all, because the old governance checks that there are no arguments, as opposed to an empty args argument.

Reply:
Ah, then we need to insert the `"args": ` level inside the `args` variable.

`**args` in the original code expands to `"next_service_identity": ...`, but in this object we need `"args": {"next_service_identity": ...}`